proto/cosmos/crypto/ledger/secp256k1/v1beta1/keys.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package cosmos.crypto.ledger.secp256k1.v1beta1;
```

Comment:
because of the `cached_pub_key` field below, I prefer to keep it as beta for now. imo `cached_pub_key` should be an implementation detail, and not exposed in proto definitions.